includes/connection/class-wc-terms.php

```diff
@@ -10,29 +10,33 @@
 
 namespace WPGraphQL\WooCommerce\Connection;
 
+use GraphQL\Type\Definition\ResolveInfo;
 use GraphQL\Error\UserError;
+use WPGraphQL\AppContext;
 use WPGraphQL\Connection\TermObjects;
+use WPGraphQL\Data\DataSource;
+use WPGraphQL\Data\Connection\TermObjectConnectionResolver;
+use WPGraphQL;
+use WP_GraphQL_WooCommerce;
```

Comment:
There's little point in a `use` statement for a class that is only using called directly once.

Reply:
Hahah! Let's agree to disagree. But one point for it is visibility.

One can see all the classes being used in a file without the need to look/scan the file. Besides, it is such a small change that it doesn't hurt.

Happy to revert it if you feel strongly about it. :)